app/scripts.babel/element_observer.js

```diff
@@ -0,0 +1,41 @@
+const HANDLER_CLASS_PREFIX = 'handler_';
+
+class ElementObserver {
+  constructor() {
+    this._handlers = [];
+    this._observer = new MutationObserver(mutations => {
+      for (const mutation of mutations) {
+        for (const node of Array.from(mutation.addedNodes)) {
+          if (!(node instanceof Element)) {
+            continue;
+          }
+
+          this._handlers.forEach((item, i) => {
+            if (node.matches(item.selector) &&
+                !node.classList.contains(HANDLER_CLASS_PREFIX + i)) {
+              // console.log('online ' + item.selector);
+              item.handler(node);
+            }
```

Comment:
I believe you need to do `node.classList.add(HANDLER_CLASS_PREFIX + handlerId)` here.